tools/glb-viewer/src/controls.js

```diff
@@ -0,0 +1,42 @@
+
+// initialize controls
+document.getElementById('play').onclick = function() {
+    viewer.play();
+};
```

Comment:
Since there's now a new `tools` folder in the root, please add `tools` to the folders eslint checks in `package.json`.

Reply:
done